### ISSUE TITLE
View history and clear it 

### ISSUE THREAD

**Issue by harmi86** [erengy/taiga#607]
Hi,

when I started using the program, I had a lot trouble to configured my torrent filters. I found the problem was that I discard all torrents via the button, so all were added to the history and so the anime were of cause not selected. I things an option would be nice that you can see the history and clear it or delete items if necessary. Maybe it should also be shown in the list view why a item is disable(with tool tip or so). I saw that you did something similar in the debug mode, because otherwise i would not found the problem, so at least the last point should not a big issue ^^

**Comment by taigalover** [comment]
Just to understand you first problem, you want to see all the torrents downloaded by Taiga then you want to delete it?

**Comment by harmi86** [comment]
No, I pressed on the button "discard all" and all were added to the history.xml in %APPDATA%\Roaming\Taiga\data\feed. After that I created my filter and was confused why they were "not working". How I know that it was because I pressed the button, so I deleted the XML-File manually and everything was working. so it was not a problem.
I thought that it would be nice to see, why a torrent is not selected, so that the user is not confused why something is not working. The functionally to clear and view the torrent history is also not bad because not all user know where they find the history-file..

**Comment by harmi86** [comment]
awesome, thx for the Implementation
